### PR TITLE
vulkan-headers: new port (submission)

### PR DESCRIPTION
**Pull request by eintw1ck**
###### Tested on
macOS 10.14.4 18E226
Xcode 10.2 10E125

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/vulkan-headers/Portfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+github.setup        KhronosGroup Vulkan-Headers sdk-1.1.101.0
+name                vulkan-headers
+version             1.1.101.0
+
+categories          graphics
+platforms           darwin
+license             Apache-2
+
+maintainers         {itanimul.li:josh @eintw1ck}
```
Comment (cjones051073) [inline graphics/vulkan-headers/Portfile]: Please add openmaintainer here.

File: graphics/vulkan-headers/Portfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+description         Vulkan header files and API registry
+long_description    Development header files for the Vulkan graphics API. See MoltenVK, and \
+                    Vulkan-Loader for related ports.
+
+homepage            https://vulkan.lunarg.com
+checksums           sha256 95c876b537356e3b7742defa45a249ad98c935dd4080c01b62c2e3d4248ed7f8
```
Comment (cjones051073) [inline graphics/vulkan-headers/Portfile]: Please add rmd160 and size checksums.

**Comment by eintw1ck** [comment]
@cjones051073 thanks for review, I have fixed the issues you mentioned.

**Comment by cjones051073** [comment]
Thanks, LGTM now.

**Pull request merged**